File: Array/easy/largest_element_in_an_array.cpp
//
// Created by santosh on 12/5/25.
//

#include<bits/stdc++.h>
using namespace std;

int largestElement(vector<int>& nums) {


    int largestElement = INT_MIN;
    for (int i = 0; i < nums.size(); i++)
    {
        if (nums[i] > largestElement)
        {
            largestElement = nums[i];
        }
    }
    return largestElement;

}

int main()
{
    vector <int> v(1000);
    iota(v.begin(), v.end(), 0);

    int largest_element=largestElement(v);
    cout<<largest_element<<endl;
}

File: Array/easy/largest_element_in_an_array.py
# Find the largest element in an array
# Created by santosh

def largest_element(nums: list[int]) -> int:
    largest = float('-inf')
    for num in nums:
        if num > largest:
            largest = num
    return largest


if __name__ == "__main__":
    nums = [6, 3, 8, 10, 16, 7, 5, 2, 9, 14]
    print("Largest element:", largest_element(nums))
